composer.json

```diff
@@ -16,7 +16,9 @@
     ],
     "require": {
         "php": "^7.1",
+        "doctrine/phpcr-bundle": "^1.4|^2.0",
```

Comment:
we do not require the persistence layer, cause basically each packaged could also work with orm. in reald it is routing and seo only.

Reply:
I removed this dependency, rebased and pushed to github again